attachments/templates/attachments/list.html

```diff
@@ -1,31 +1,29 @@
-{% load bootstrap %}
-
 {% if session.uploads.exists %}
-<table class="table table-condensed attachment-files-table">
-    <thead>
-        <tr>
-            <th scope="col">File</th>
-            <th scope="col">Size</th>
-            <th scope="col">Remove</th>
-        </tr>
-    </thead>
-    {% for upload, property_form in session.upload_forms %}
-        <tbody class="upload-item">
-            <tr>
-                <td>{{ upload.file_name }}</td>
-                <td>{{ upload.file_size|filesizeformat }}</td>
-                <td>
-                    <a href="{% url 'delete-upload' session_id=session.uuid upload_id=upload.pk %}" class="delete-upload">
-                        {% block delete-upload %}Remove <span class="sr-only">{{ upload.file_name }}</span>{% endblock %}
-                    </a>
-                </td>
-            </tr>
-            {% if property_form %}
-                <tr class="properties">
-                    <td colspan="3">{% include "attachments/form.html" with form=property_form %}</td>
-                </tr>
-            {% endif %}
-        </tbody>
-    {% endfor %}
-</table>
+	<table class="table table-condensed attachment-files-table">
+	    <thead>
+	        <tr>
+	            <th scope="col">File</th>
+	            <th scope="col">Size</th>
+	            <th scope="col">Remove</th>
+	        </tr>
+	    </thead>
+	    {% for upload, property_form in session.upload_forms %}
+	        <tbody class="upload-item">
+	            <tr>
+	                <td>{{ upload.file_name }}</td>
+	                <td>{{ upload.file_size|filesizeformat }}</td>
+	                <td>
+	                    <a href="{% url 'delete-upload' session_id=session.uuid upload_id=upload.pk %}" class="delete-upload">
+	                        {% block delete-upload %}Remove <span class="sr-only">{{ upload.file_name }}</span>{% endblock delete-upload %}
+	                    </a>
+	                </td>
+	            </tr>
+	            {% if property_form %}
+	                <tr class="properties">
+	                    <td colspan="3">{% include {% block form %}"attachments/form.html"{% endblock form %} with form=property_form %}</td>
```

Comment:
I don't think nesting a block tag inside of an include tag is valid django.  Include can take context variables, but I don't think it can do blocks like that